biothings/web/query/pipeline.py

```diff
@@ -100,52 +106,47 @@ async def _(*args, **kwargs):
         ) as exc:
             logger.debug("QueryPipelineInterrupt: %s", exc.data)
             raise QueryPipelineInterrupt(exc.data)
```

Comment:
Linters giving the suggestion we can attach the original context to the exception for all of these if we think it's relevant.
`raise-missing-from: Consider explicitly re-raising using 'raise QueryPipelineInterrupt(exc.data) from exc'`

Reply:
Understood that makes sense. What do you think would entail a refactor for our exception handling if we were to make a ticket about it?

Reply:
I think it's a great idea creating a ticket for this!